spec/views/medicaid/income_other_income_type/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+describe "medicaid/income_other_income_type/edit.html.erb" do
+  before do
+    controller.singleton_class.class_eval do
+      def current_path
+        "/steps/medicaid/income-other-income-type"
+      end
+
+      def current_member
```

Comment:
defining `current_member` up here but `member` in the test itself feels kind of confusing to me.

Reply:
`current_member` is up here so that the view has access to it. It's a gateway to what we create in the setup phase.

It's weird, I agree, but it's a necessity to get this to function.  Now, if we had *one* object passed through from the controller to the view then this wouldn't be necessary. Like `@step.current_member`. It's an interesting smell that we provide access to this many objects in the view. View test is a nice way to suss those out.